packages/examples/.storybook/preview.ts

```diff
@@ -0,0 +1,15 @@
+import type { Preview } from "@storybook/react";
+
+const preview: Preview = {
+  parameters: {
+    actions: { argTypesRegex: "^on[A-Z].*" },
+    controls: {
+      matchers: {
+        color: /(background|color)$/i,
+        date: /Date$/,
+      },
+    },
+  },
+};
+
+export default preview;
```

Comment:
beep boop beep

No idea what this all is for 😄

Reply:
No worries [main.ts](https://storybook.js.org/docs/react/configure/overview#configure-your-storybook-project) and [preview.ts](https://storybook.js.org/docs/react/configure/overview#configure-story-rendering) configure your Storybook. The first one is responsible for more low level config (what UI framework, what features, addons, etc.), while in preview one can configure global decorators and parameters. The current `preview.ts` was just autogenerated by Storybook, I will see if it makes sense to make it even simpler :)

Reply:
Nah no worries, I was just mostly having a bit of fun 😊